spec/requests/api/conditions_spec.rb

```diff
@@ -18,6 +18,91 @@ def assign_conditions_to(resource)
     resource.conditions = Condition.all
   end
 
+  context "Condition CRUD" do
+    let(:sample_condition) do
+      {
+        :name        => "name",
+        :description => "description",
+        :expression  => {"=" => {"field" => "ContainerImage-architecture", "value" => "dsa"}},
+        :towhat      => "ExtManagementSystem",
+        :modifier    => "allow"
+      }
+    end
+    let(:condition) { FactoryGirl.create(:condition) }
+    let(:condition_url) { conditions_url(condition.id) }
+    let(:conditions) { FactoryGirl.create_list(:condition, 2) }
+
+    it "forbids access to conditions without an appropriate role" do
```

Comment:
@alongoldboim could you add authorization tests for each of the new actions, create, edit and delete as noted by @imtayadeway thanks.